[2024-10-03T19:51:22.405+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T19:51:22.415+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T14:21:07.180648+00:00 [queued]>
[2024-10-03T19:51:22.421+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T14:21:07.180648+00:00 [queued]>
[2024-10-03T19:51:22.421+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-03T19:51:22.450+0530] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): validate_data> on 2024-10-03 14:21:07.180648+00:00
[2024-10-03T19:51:22.455+0530] {standard_task_runner.py:72} INFO - Started process 21883 to run task
[2024-10-03T19:51:22.459+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_ingestion_dag', 'validate_data', 'manual__2024-10-03T14:21:07.180648+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmps_zg6rnp']
[2024-10-03T19:51:22.462+0530] {standard_task_runner.py:105} INFO - Job 102: Subtask validate_data
[2024-10-03T19:51:22.509+0530] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T14:21:07.180648+00:00 [running]> on host Nithesh-Kumar.localdomain
[2024-10-03T19:51:22.601+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_ingestion_dag' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-03T14:21:07.180648+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-03T14:21:07.180648+00:00'
[2024-10-03T19:51:22.603+0530] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T19:51:22.699+0530] {file_data_context.py:222} INFO - FileDataContext loading fluent config
[2024-10-03T19:51:22.701+0530] {config.py:187} INFO - Loading 'datasources' ->
[]
[2024-10-03T19:51:22.806+0530] {validator.py:1467} INFO - 	28 expectation(s) included in expectation_suite.
[2024-10-03T19:51:22.999+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T19:51:22.999+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T19:51:23.000+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:00, 2235.17it/s]
[2024-10-03T19:51:23.032+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:01, 61.47it/s]  
[2024-10-03T19:51:23.032+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:00, 90.85it/s]
[2024-10-03T19:51:23.063+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:01, 47.01it/s]
[2024-10-03T19:51:23.069+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 243.26it/s]
[2024-10-03T19:51:23.095+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 176.77it/s]
[2024-10-03T19:51:23.127+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 571.03it/s]
[2024-10-03T19:51:23.127+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 571.03it/s]
[2024-10-03T19:51:23.140+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 571.03it/s]
[2024-10-03T19:51:23.140+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 571.03it/s]
[2024-10-03T19:51:23.140+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 517.68it/s]
[2024-10-03T19:51:23.500+0530] {logging_mixin.py:190} INFO - Data quality check passed.
[2024-10-03T19:51:24.289+0530] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T19:51:24.294+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T19:51:24.295+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_dag, task_id=validate_data, run_id=manual__2024-10-03T14:21:07.180648+00:00, execution_date=20241003T142107, start_date=20241003T142122, end_date=20241003T142124
[2024-10-03T19:51:24.361+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T19:51:24.390+0530] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-03T19:51:24.409+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
